sass/tji/_buttons.scss

```diff
@@ -44,4 +44,20 @@
     opacity: 1.0;
     @include transition(opacity, 1s);
   }
+}
+
+.tji-form-submit {
```

Comment:
can you explain why you chose to use !important. Usually that's not a great practice because it makes the css rigid -- and difficult to change.

Reply:
Agreed, I am running into conflict with styles we have for general type=
submit". This is redundant but I wasn't sure if I should take care of this here or when we do our overhaul for the next iteration of the site.